sharepoint/sharepoint-ps/sharepoint-online/Set-SPOMultiGeoExperience.md

```diff
@@ -14,7 +14,7 @@ ms.reviewer:
 
 ## SYNOPSIS
 
-.
+Used to set a geo location into SPO mode.
```

Comment:
Let's rephrase this, something like:

"Used to add SharePoint Online to the multi-geo instance."

Reply:
Hello Trevor, the sentence is taken from here 
https://docs.microsoft.com/en-us/microsoft-365/enterprise/enabling-sp-multigeo-satellite-geolocation?view=o365-worldwide#to-set-a-geo-location-into-spo-mode

Do you still feel it needs a re-phrase? Thank you